sample/src/main/scala/SampleExports.scala

```diff
@@ -0,0 +1,28 @@
+package sample
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation._
+
+object SampleExports {
+  @JSExportTopLevel("immutableField")
+  val immutableField: String = "my immutable field value"
+
+  @JSExportTopLevel("mutableField")
+  var mutableField: Int = 42
+
+  @JSExportTopLevel("simpleFunction")
+  def simpleFunction(x: Int): Int =
+    x * x
+
+  @JSExportTopLevel("functionWithRest")
+  def functionWithRest(x: Int, rest: Int*): Int =
+    x * rest.sum
+
+  @JSExportTopLevel("SimpleClass")
+  class SimpleClass(val foo: Int) extends js.Object
+
+  @JSExportTopLevel("SimpleObject")
+  object SimpleObject extends js.Object {
+    val bar: String = "the bar field"
+  }
```

Comment:
It would be nice to keep `Sample.scala` and `run.mjs` as it is.
What do you think about having these `JSExportTopLevel`s in `Test` scope of `sample` project like `SampleTest.scala` and run `test-exports.mjs` using the `./sample/target/scala-2.12/sample-test-fastopt/main.mjs` instead?

Reply:
We can merge this PR with this, and fix it later of course ;)

Reply:
Ah yes, you're right. That wasn't a great idea. Using the Test scope doesn't work because it would launch the test bridge as part of its `start` function, and we don't want that either.

I instead added a bit more infrastructure in `tests/test` so that we can test the resulting module exports of a test in `test-suite`. Is that better?

Reply:
> I instead added a bit more infrastructure in tests/test so that we can test the resulting module exports of a test in test-suite. Is that better?

Yeah, this is awesome, thanks!